Makefile

```diff
@@ -1,24 +1,25 @@
 S3_CP_ARGS=aws s3 cp --acl public-read
+RELEASE_VERSION?=
+
+S3_BUCKET_PATH=s3://observe/cloudformation/
+S3_EXISTING_VERSIONS=$(shell aws s3 ls $(S3_BUCKET_PATH) | grep $(RELEASE_VERSION) | wc -l)
 
 .PHONY: cloudformation
 cloudformation:
```

Comment:
Would be good to give this a descriptive name, e.g. `copy_s3`

Reply:
resolved in [e93dc8f](https://github.com/observeinc/cloudformation-aws-collection/pull/35/commits/e93dc8f8fbfd74a79452f2fa9adadc7dadd41c64)

Reply:
resolved in [e93dc8f](https://github.com/observeinc/cloudformation-aws-collection/pull/35/commits/e93dc8f8fbfd74a79452f2fa9adadc7dadd41c64)